back/db/migrate/20200527094026_add_ordering_to_projects_topics.rb

```diff
@@ -1,10 +1,18 @@
 class AddOrderingToProjectsTopics < ActiveRecord::Migration[6.0]
-  def change
+  class StubProjectsTopic < ActiveRecord::Base
+    self.table_name = 'projects_topics'
+  end
+
+  def up
     add_timestamps :projects_topics
     add_column :projects_topics, :ordering, :integer
 
-    ProjectsTopic.order(:created_at).each.with_index do |projects_topic, index|
+    StubProjectsTopic.order(:created_at).each.with_index do |projects_topic, index|
       projects_topic.update_column :ordering, index
     end
   end
+
+  def down
+   # projects_topics :ordering, :integer
+  end
```

Comment:
I was trained to never change existing migrations. However, in this case the original migration referenced a class, and Rails would complain this class does not exist (which it doesn't at this point) when running `db:migrate`. Stubbing the class solves this problem.
Thanks to @alexander-cit for this solution.

Reply:
I think for `down` the ordering and timestamps columns should be removed.